app/lib/govuk_elements_form_builder/form_builder.rb

```diff
@@ -227,13 +228,19 @@ def hint?(attribute, options)
     end
 
     def hint_message(attribute, options)
+      return options[:hint] if options[:radio_button]
+
```

Comment:
The effect of this change is to remove the option of putting translations into `helpers.hint` if they are radio buttons. I wonder if this is a better alternative:
```ruby
def hint_message(attribute, options)
  return options[:hint] if option[:hint].present?
  
  translation_path = ['helpers.hint', @object_name, attribute]
  translation_path << options[:value] if options[:radio_button]
  I18n.translate(translation_path.join('.'), default: nil)
end
```
That would then allow the translations for a `bar` attribute of `foo` with radios `yes` and `no` to have it hints at:
```yaml
en:
  helpers:
    hint:
      foo:
        bar:
          yes: Oh yes
          no: Hell no
```

Reply:
If `options[:radio_button]` returned the current `value`, the code could change to:
```ruby
translation_path << options[:radio_button] if options[:radio_button]
```
Or you could do:
```ruby
I18n.translate(
  ['helpers.hint', @object_name, attribute, options[:radio_button]].compact.join('.'),
  default: true
)
```

Reply:
The problem with
```yaml
en:
  helpers:
    hint:
      foo:
        bar:
          yes: Oh yes
          no: Hell no
```
Is that the hint message of the radio button collection (the one above the options) will now be the hash:
```ruby
{ yes: 'Oh yes', no: 'Hell no' }
```

Reply:
Because that's how we use it for now:
i.e. : 
```yaml
en:
  helpers:
    hint:
      user:
        name: Enter your name
```

Reply:
OK